packages/wonder-blocks-color/util/utils.test.js

```diff
@@ -0,0 +1,190 @@
+// @flow
+import {fade, mix} from "./utils.js";
+
+const INVALID_COLORS = ["#NOTVALID", "#12345", "#12", "#ABCDEFG", ""];
+
+describe("mix", () => {
+    test("color is not a string, throws", () => {
+        // Arrange
+        const color = ((null: any): string);
+
+        // Act
+        const underTest = () => mix(color, "#FFF");
+
+        // Assert
+        expect(underTest).toThrowErrorMatchingSnapshot();
```

Comment:
I don't think there's much benefit in using snapshots for testing this.
```
expect(underTest).toThrow(new Error("Failed to parse color: #FFF"))
```
isn't much longer and you don't have to go look at another file to see what error we were expecting.

Reply:
That's the point of using a snapshot. It avoids having to go copying strings around. The error snapshots are there for exactly this kind of thing. I think we should use the tools we have.

Also, it's a test. We don't need to go see what it was expecting unless it fails.

Reply:
Okay.